src/main/java/ClassApp.java

```diff
@@ -0,0 +1,11 @@
+public class ClassApp {
+
+    public static void main(String[] args) {
+        ClassLesson5 cl = new ClassLesson5();
+        cl.myNumbers();
+
+        int number2 = 10;
+        int result = cl.myNumbers() + number2;
```

Comment:
This is not from homework. It's not a problem but do not push changes which is not required in HW